addons/TreasurePool/TreasurePool.lua

```diff
@@ -0,0 +1,160 @@
+--[[Copyright © 2015, Kenshi
```

Comment:
Yes, it has been *that* long since you requested this :D Please change this to 2016.